src/js/consts/applicationUrls.js

```diff
@@ -5,7 +5,7 @@
 export const CONTEXT_PATH = window.CONTEXT_PATH ?? '/openboxes';
 
 const DASHBOARD_URL = {
-  base: '/dashboard',
+  base: `${CONTEXT_PATH}/dashboard`,
```

Comment:
what I meant by my comment is that this part should not be necessary in my opinion - it should be enough to write `history.push('/dashboard')`